clang/docs/ReleaseNotes.rst

```diff
@@ -259,6 +259,10 @@ AIX Support
 WebAssembly Support
 ^^^^^^^^^^^^^^^^^^^
 
+The -mcpu=generic configuration now enables nontrapping-fptoint, multivalue,
+reference-types, and bulk-memory.These proposals are standardized and available
+in all major engines.
```

Comment:
Could you also mention that enabling multivalue here just means enabling the language feauture, and not changing the ABI?

Reply:
Done